### DISCUSSION TITLE
get VRF prefixes

### DISCUSSION THREAD

**Discussion by AIhmed** [netbox-community/ansible_modules#1017] [General]
I want to query all prefixes in the vrf. When i the run the following query i get all the prefixes in netbox:

`vrf_prefixes: "{{ query('netbox.netbox.nb_lookup', 'prefixes', api_filter='vrf_name=' + vrf_name, api_endpoint=netbox_url, token=netbox_token) }}"`

vrf is an object that looks like this:

`"vrf": {
                    "display": "",
                    "id": ,
                    "name": "",
                    "rd": "",
                    "url": ""
                }`
I only want to do the lookup with the name of the vrf, what query should i run to get all the prefixes inside a specifique vrf?

**Comment by sc68cal** [comment]
https://github.com/netbox-community/ansible_modules/issues/539#issuecomment-1586274159
